pkgs/by-name/hy/hyprprop/package.nix

```diff
@@ -0,0 +1,50 @@
+{ fetchFromGitHub
+, socat
+, jq
+, lib
+, makeWrapper
+, slurp
+, stdenv
+, hyprland
+, hyprevents
+, bash
+}:
+
+stdenv.mkDerivation {
+  # pkill is a dependency here.. but its in PATH and its hyprevents (not hyprprop) that needs to access it...
```

Comment:
```suggestion
```
Not sure if we need to wrap [`procps`](https://gitlab.com/procps-ng/procps) for both. Can someone else chime in?